.github/dependabot.yml

```diff
@@ -10,4 +10,4 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - "area/dependencies"
+      - "kind/dependencies"
```

Comment:
Will dependabot still add the "rust" label? Is this specifying a label for it to add to PRs?

Reply:
No dependebot won't and hasn't in a while